layouts/partials/breadcrumbs.html

```
@@ -0,0 +1,25 @@
+{{- if (.Param "ShowBreadCrumbs") -}}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)